pyproject.toml

```diff
@@ -16,6 +16,8 @@ scrapelib = "^1.2"
 lxml = "^4.4"
 openstates = "^5.3.0"
 cssselect = "^1.1.0"
+airtable-python-wrapper = "0.15.1"
```

Comment:
let's save this for the airtable PR

Reply:
91ca446b8785b2ffc3809861a5d859c3bf07b1a7

good catch. I had to manually extract this PR from the bigger airtable one and missed that.